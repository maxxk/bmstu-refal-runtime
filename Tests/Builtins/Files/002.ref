$ENTRY Go
{
	= <Open 'w' 20 '../../Tests/Files/test1.write.tmp'>
}
