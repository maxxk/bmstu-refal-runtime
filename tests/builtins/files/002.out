[Builtins]: Bad file descriptor! Descriptor must be in range [1,20)! Descr: 20
